### PR TITLE
Feature/arc 1344

### PR DESCRIPTION
**Pull request by silkederudder-shd**
### Admin
(Ben je leerkracht? item should not be active on these screens, I only added this to demonstrate it)
admin-desktop
admin-mobile

### CP
cp-desktop
cp-mobile-1
cp-mobile-2

### Logged out
logout-desktop
logout-mobile

### CODE REVIEW

File: src/modules/navigation/components/Navigation/Navigation.consts.tsx [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
+	NavigationInfo,
+	NavigationPlacement,
+} from '@navigation/services/navigation-service/navigation.types';
 import { Icon, IconName, IconNamesLight } from '@shared/components';
 import { ROUTE_PARTS, ROUTE_PREFIXES, ROUTES } from '@shared/const';
 import { tText } from '@shared/helpers/translate';
 import { Breakpoints } from '@shared/types';
 import { VisitorSpaceInfo } from '@visitor-space/types';
 
+export enum NAVIGATION_DROPDOOWN {
```
Comment (tourlbr) [inline src/modules/navigation/components/Navigation/Navigation.consts.tsx:20]: PP: add enum to types file

**Review by tourlbr — APPROVED** [review]

**Review by tourlbr — APPROVED** [review]

**Review by WardVerc1 — APPROVED** [review]

**Comment by bertyhell** [comment]
"Zoeken naar aanbieders" should link to "/bezoek"

image


Going to the search page, crashes the app. You probably need to use ?.replaceAll since the maintainer query param doesn't always have a value

image

**Comment by bertyhell** [comment]
The mobile view of the logged out homepage doesn't look completely correct:
image

**Review by bertyhell — CHANGES_REQUESTED** [review]
see global remarks

**Comment by silkederudder-shd** [comment]
> see global remarks

@bertyhell, see https://meemoo.atlassian.net/browse/ARC-1376. The white line had to be removed on both the home and search page.

**Comment by silkederudder-shd** [comment]
> 

image

**Comment by bertyhell** [comment]
The "bezoek een aanbieder should not be underlined when you're at the homepage:
image

**Review by bertyhell — CHANGES_REQUESTED** [review]
see global remark

File: src/modules/visitor-space/const/index.tsx [hunk 2]
```diff
@@ -53,7 +53,7 @@ export const VISITOR_SPACE_QUERY_PARAM_CONFIG = {
 	// Filters
 	format: withDefault(StringParam, VISITOR_SPACE_QUERY_PARAM_INIT.format),
 	[SEARCH_QUERY_KEY]: ArrayParam,
-	[VisitorSpaceFilterId.Maintainer]: StringParam,
+	[VisitorSpaceFilterId.Maintainer]: withDefault(StringParam, ''),
```
Comment (bertyhell) [inline src/modules/visitor-space/const/index.tsx:56]: why is this needed? i would think we just omit the maintainer query param if it is empty?

**Review by bertyhell — APPROVED** [review]

**Pull request merged**